src/tests/Microsoft.PowerFx.Interpreter.Tests/LanguageTest.cs

```diff
@@ -63,6 +64,31 @@ public void GetLanguageForInvariantCulture()
             ConfigTests.RunOnIsolatedThread(_defaultCulture, TestDefaultCulture);
         }
 
+        [Theory]
+        [InlineData("1234,5678", "#.##0,00", "1.234,57", "vi-VN")]
+        [InlineData("1234,5678", "#.##0,00", "1.234,57", "pt-BR")]
+        [InlineData("1234,5678", "# ##0,00", "1 234,57", "fr-FR")]
```

Comment:
>1 23 [](http://example.com/codeflow?start=46&length=4)

Is the space between 1 and 2 the U+202F character? If that's the case it would be better to be explicit and define the string as "1\u202f234,57".

Reply:
update